fvt/storage/storage_suite_test.go

```diff
@@ -0,0 +1,135 @@
+// Copyright 2023 IBM Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package storage
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	. "github.com/kserve/modelmesh-serving/fvt"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestStorage(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Storage Suite")
+}
+
+func createFVTClient() {
```

Comment:
This looks to be copied as-is from the predictor suite. It could be moved to `fvt/utils.go` to have the code shared (and probably should be renamed to something like `initializeFVTSuite()`.

Reply:
it's kind of the same code in all 3 suites. I can refactor it

Reply:
I did the refactoring for all 3 suites